avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastStringableTest.java

```diff
@@ -137,6 +140,31 @@ public void serializeStringableFields() throws URISyntaxException, MalformedURLE
     }
   }
 
+  @Test(groups = {"deserializationTest"}, dataProvider = "SlowFastDeserializer")
+  public void javaStringPropertyTest(Boolean whetherUseFastDeserializer) throws IOException {
```

Comment:
Can we also check in the code gen of this test for a better review?

Reply:
If you wish to include more tests in the code gen under source control, you can use `FastSerdeTestsSupport#getCodeGenDirectory()` and an example of its usage in `FastGenericDeserializerGeneratorTest`.

If you choose to add more stuff to the code gen under source control, then you should also split that into an extra commit (one to add the new code gen, one to modify the meta-code, and another one for the modified code gen) otherwise we can't see the diff if it's all in one commit...

Reply:
Great idea, I like the way how to store the generated code!